client/src/pages/hallOfFame/HallOfFame.tsx

```diff
@@ -1,11 +1,24 @@
 import FlavorThumbnail, { Flavor } from "../../components/FlavorThumbnail";
 import styles from "./hall-of-fame.module.css";
+import axios from "axios";
 
-const flavors: Flavor[] = Array.from({ length: 22 }, () => ({
-  name: "Vanilla",
-  price: 0,
-  image: "/strawberry-icecream.jpg",
-}));
+const hofFlavors = await axios
+  .get("http://localhost:3003/api/hof")
+  .then((result) => {
```

Comment:
This .then is unnecessary - the result will be put into hofFlavors. Then below, you can do hofFlavors.data.forEach. Or, you can do it like this:

const response = await axios.get("url")
let hofFlavors = response.data

Checkout the section on await here:
https://javascript.info/async-await

Reply:
Josh removed it, thanks!